test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -60,6 +60,33 @@ def create_mom_resources(self, resc_name,
                                         validate=True)
         self.scheduler.add_resource(resc_name)
 
+        return fp
+
+    def create_mult_mom_resources(self, resc_name,
+                                  resc_type, resc_flag,
+                                  script_body):
+        """
```

Comment:
I'd merge the two create functions as @anamikau suggested.  I don't think it is hard to pass a list of one item instead of the item itself.  If you do want to keep the functions separate, at least have this function call the first one.